### DISCUSSION TITLE
Settings for parallel mode

### DISCUSSION THREAD

**Discussion by DanielFNG** [coin-or/rbfopt#47] [Q&A] [answered]
Hello, 

I note that in the readme it says that the default settings are optimised for running in serial. I don't seem to be able to get access to the mailing list, so I wondered if anyone could share some optimised settings for running in parallel here? 

Thanks,

Daniel

**Comment by gnannicini** [comment, marked as answer]
There are no specific recommended settings for the parallel mode, as they generally depend on the number of CPU/threads you are planning to use.

Generally, you may want to increase the num_global_searches so that it scales with the number of CPUs (e.g., num_cpus/2 or even just num_cpus). If there are too many restarts, you may want to change discarded_window_size or max_fraction_discarded. All of this is very problem-dependent, so you may need some empirical testing.
